model/testing/src/icon4py/model/testing/fixtures/stencil_tests.py

```diff
@@ -24,74 +24,100 @@
     "simple",
     "icon_regional",
     "icon_global",
-    "icon_benchmark",
+    "icon_benchmark_regional",
+    "icon_benchmark_global",
 )
 
 
-def _get_grid_from_preset(
+def _get_grid_manager_from_preset(
     grid_preset: str,
     *,
     num_levels: int = DEFAULT_NUM_LEVELS,
     backend: gtx_typing.Backend | None = None,
-) -> base_grid.Grid:
+) -> gm.GridManager | None:
     match grid_preset:
         case "icon_regional":
             return grid_utils.get_grid_manager_from_identifier(
                 definitions.Grids.MCH_CH_R04B09_DSL,
                 num_levels=num_levels,
                 keep_skip_values=False,
                 backend=backend,
-            ).grid
+            )
         case "icon_global":
             return grid_utils.get_grid_manager_from_identifier(
                 definitions.Grids.R02B04_GLOBAL,
                 num_levels=num_levels,
                 keep_skip_values=False,
                 backend=backend,
-            ).grid
-        case "icon_benchmark":
+            )
+        case "icon_benchmark_regional":
             return grid_utils.get_grid_manager_from_identifier(
                 definitions.Grids.MCH_OPR_R19B08_DOMAIN01,
-                num_levels=80,  # default benchmark size in ICON Fortran
+                num_levels=num_levels,
```

Comment:
setting this here was on purpose, such that local benchmarking matches ci setup by default.